docs/unumpy_using.rst

```diff
@@ -0,0 +1,35 @@
+.. currentmodule:: unumpy
+
+Using ``unumpy``
+================
+
+``unumpy`` is an in-progress mirror of the NumPy API which allows the user
+to dynamically switch out the backend that they are using. It also allows
+auto-selection of the backend based on the arguments passed into a function.
+
```

Comment:
Before listing supported functions/objects, perhaps a bit more context here. Something like: "note: currently coverage is very incomplete, we are focusing on supporting one or a few of each *kind* of object - regular functions, ufuncs, classes and methods like `np.ufunc` and `np.ufunc.reduce`, `dtypes`, etc.".